lib/components/primitive-components/Via.ts

```diff
@@ -55,6 +69,32 @@ export class Via extends PrimitiveComponent<typeof viaProps> {
     })
   }
 
+  _getLayers(): LayerRef[] {
+    const { fromLayer = "top", toLayer = "bottom" } = this._parsedProps
+    if (fromLayer === toLayer) return [fromLayer]
+    return [fromLayer, toLayer]
```

Comment:
this is technically wrong, just make a // TODO calculate layers inbetween top and bottom using layer count